internal/fositestoragei/fosite_sotrage_interface.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2021 the Pinniped contributors. All Rights Reserved.
```

Comment:
Nit: the name of this file looks like a typo.

Reply:
Fixed in 5c62a9d